src/utils/rfqm_health_check.ts

```diff
@@ -1,8 +1,18 @@
+import { BigNumber } from '@0x/utils';
 import { Producer } from 'sqs-producer';
 
-const SQS_QUEUE_SIZE_DEGRADED_THRESHOLD = 10; // More messages sitting in queue than this will cause a DEGRADED issue
-const SQS_QUEUE_SIZE_FAILED_THRESHOLD = 20; // More messages sitting in queue than this will cause a FAILED issue
+import { ETH_DECIMALS } from '../constants';
+import { RfqmWorkerHeartbeatEntity } from '../entities';
 
+const SQS_QUEUE_SIZE_DEGRADED_THRESHOLD = 10; // (minutes) More messages sitting in queue than this will cause a DEGRADED issue
+const SQS_QUEUE_SIZE_FAILED_THRESHOLD = 20; // (minuteds) More messages sitting in queue than this will cause a FAILED issue
+
+const RECENT_HEARTBEAT_AGE_THRESHOLD = 5; // (minutes) Heartbeats older than this will produce a DEGRADED issue. A FAILED issue is produced if NO heartbeats are newer than this.
+
+const BALANCE_FAILED_THRESHOLD = 0.04; // (eth) If NO worker has a balance higher than this, a FAILED issue gets created.
+const BALANCE_DEGRADED_THRESHOLD = 0.25; // (eth) If a worker's balance is lower than this, a DEGRADED issue gets created.
```

Comment:
can this be `BALANCE_FAILED_THRESHOLD * 4`?

Reply:
sure